test/e2e/utils/recordedit-helpers.js

```diff
@@ -321,7 +347,51 @@ exports.testPresentationAndBasicValidation = function(tableParams, isEditMode) {
                     });
                 });
             }
+            
+            if (jsonCols.length > 0) {
+                describe("JSON fields, ", function () {
+                    it("should show textarea input for JSON datatype and then set the value", function() {
+                        jsonCols.forEach(function(c) {
+                        chaisePage.recordEditPage.getTextAreaForAcolumn(c.name, recordIndex).then(function(jsonTxtArea) {
+                            if (jsonTxtArea) {
+                                expect(true).toBeDefined();
```

Comment:
@jrchudy This is the way all the previous recordedit-helper test cases were written before and I didn't change them when I was refactoring it. @nitishksahu just used the same pattern of testing in here.
```javascript
if (exists) {
// do other tests knowing the field exists
} else {
// return error that the field is does not exist
}
```
We can change them to just have the `toBeDefined` as the first expect and remove `else` clause. This pattern has been used heavily in this file.